### DISCUSSION TITLE
ERR TLS Certificate failed to provision

### DISCUSSION THREAD

**Discussion by kimtaeh3** [loophole/cli#223] [Q&A]
Hi Guys,

For some reason, 1 out of 5 loophole I've created does not work - even if I try to create the new service.

when I run "systemctl status servicename.service"

Mar 26 00:25:04 nginx-virtual-machine start.sh[13589]: Loophole - End to end TLS encrypted TCP communication between you and your clients
Mar 26 00:25:06 nginx-virtual-machine start.sh[13589]: [880B blob data]
Mar 26 00:25:06 nginx-virtual-machine start.sh[13589]: [44B blob data]
Mar 26 00:25:06 nginx-virtual-machine start.sh[13589]: [970B blob data]
Mar 26 00:25:07 nginx-virtual-machine start.sh[13589]: Forwarding https://servicename.loophole.site -> http://127.0.0.1:80
Mar 26 00:25:07 nginx-virtual-machine start.sh[13589]: Press CTRL + C to stop the service
Mar 26 00:25:07 nginx-virtual-machine start.sh[13589]: Logs:
Mar 26 00:25:07 nginx-virtual-machine start.sh[13589]: 12:25AM INF Awaiting connections...
Mar 26 00:25:07 nginx-virtual-machine start.sh[13589]: 12:25AM INF Succeeded to accept connection over HTTPS
Mar 26 00:25:08 nginx-virtual-machine start.sh[13589]: 12:25AM ERR TLS Certificate failed to provision. Will be obtained with first request made by any client, therefore first execution may be slower.

**Comment by gnikolopoulos** [comment]
Same here, only it happened to all my tunnels (3). using different OS, machines and network did not help either.

**Comment by 0x7f** [comment]
Very sorry about the issue. Should be working again.

We still did not find time yet to fix it. The API service hangs every couple of weeks. A simple restart fixes it each time. Until it hangs again. A permanent fix will be coming soon ™️ ;)

**Reply by gnikolopoulos** [reply]
Yep, confirmed working on all 3 tunnels. Would be nice to have the option to drop the certificate altogether if something similar happens so people can be up and running...
Or you can set up a cron job to restart the system once a week (Joke...)
